web/modules/custom/dept_migrate/modules/dept_migrate_group_users/dept_migrate_group_users.services.yml

```diff
@@ -0,0 +1,6 @@
+services:
+  dept_migrate_group_users.eventsubscriber.post:
+    class: 'Drupal\dept_migrate_group_users\EventSubscriber\UserEntityMigrateSubscriber'
```

Comment:
I've split these into two different handlers for clarity over efficiency. It might well be possible to crunch down to a single event subscriber and hand off the migrate row/entity to the required callback based on entity type... but there's probably not a lot in it in terms of maintenance overhead right now. Open to feedback!